samza-core/src/main/java/org/apache/samza/storage/StorageRecovery.java

```diff
@@ -101,7 +98,7 @@ private void setup() {
     getContainerModels();
     getChangeLogSystemStreamsAndStorageFactories();
     getChangeLogMaxPartitionNumber();
-    getContainerStorageManager();
+    getContainerStorageManagers();
```

Comment:
No, it seems the StorageRecovery tool has always assumed stores across multiple containers (see the getContainerModels() method in it above).

It was working earlier because CSM was earlier being handed taskStores, consumers, etc. Now CSM spawns these itself for 1 container.